plugin/advisor/pg/advisor_comment_convention.go

```diff
@@ -0,0 +1,97 @@
+package pg
+
+// Framework code is generated by the generator.
+
+import (
+	"fmt"
+
+	"github.com/bytebase/bytebase/plugin/advisor"
+	"github.com/bytebase/bytebase/plugin/advisor/db"
+	"github.com/bytebase/bytebase/plugin/parser/ast"
+)
+
+var (
+	_ advisor.Advisor = (*ColumnCommentConventionAdvisor)(nil)
+	_ ast.Visitor     = (*columnCommentConventionChecker)(nil)
+)
+
+func init() {
+	advisor.Register(db.Postgres, advisor.PostgreSQLCommentConvention, &ColumnCommentConventionAdvisor{})
+}
+
+// ColumnCommentConventionAdvisor is the advisor checking for column comment convention.
+type ColumnCommentConventionAdvisor struct {
```

Comment:
After https://github.com/bytebase/bytebase/pull/2874/files#r985239847, we'd better change the name and comment to `CommentConventionAdvisor`, instead of `ColumnCommentConventionAdvisor`

Reply:
@Kikkon Hi~ It seems that you forgot to rename this.

Reply:
Sorry, it's now resolved 🥲